inspirehep/records/marshmallow/literature/base.py

```diff
@@ -305,11 +305,12 @@ def get_bookautocomplete(self, record):
 
     @pre_dump
     def populate_ui_display(self, data):
-        data["_ui_display"] = deepcopy(data)
+        data["_ui_display"] = copy(data)
```

Comment:
This is needed because `_ui_display` is defined as a `Nested` field in the schema, so it has to be there from the start. I think it's better to use a `Method` there that calls the UI serializer on the record explicitly in order to avoid these hacks.